poverty_info.rb

```diff
@@ -0,0 +1,47 @@
+require 'pry'
+
+STATE_FILES = { "Washington" => "povWA.txt", "South Dakota" => "povSD.txt"}
+
+def print_child_poverty_info(state)
```

Comment:
I think it would follow single-responsibility principle if you had separate methods that would calculate the min and max for your files and use those to print out the data